build/PackageDiffIgnore.xml

```diff
@@ -226,6 +226,8 @@
 			</Types>
 
 			<Methods>
+				<Member fullName="Windows.UI.Xaml.Controls.CalendarDatePickerDateChangedEventArgs..ctor()"
```

Comment:
The ignore set should contain the full method specification:
```suggestion
				<Member fullName="System.Void Windows.UI.Xaml.Controls.CalendarDatePickerDateChangedEventArgs..ctor()"
```

Reply:
Thanks. This is so simple that I cannot see it myself :)

One more: what means " Removed method System.Void Windows.UI.Xaml.Controls.CalendarDatePicker.add_DateChanged"?
Is this some auto generated method, similar to CalendarDatePickerDateChangedEventArgs constructor, to be placed in PackageDiffIgnore.xml ?